__tests__/api.test.js

```diff
@@ -397,4 +397,68 @@ describe("GET GET /api/articles (sorting queries)", () => {
                 expect(body.msg).toBe("invalid request")
             })
     })
+})
+
+describe("GET /api/articles (topic queries)", () => {
+    test("200: Responds with an array of article/articles filtered by topic query mitch", () => {
+        return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles.length >= 1).toBe(true)
+                body.articles.forEach((article) => {
+                    expect(article).toMatchObject({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        topic: "mitch",
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(String),
+                        article_id: expect.any(Number),
+                        created_at: expect.any(String)
+                    })
+                })
+            })
+
+    })
+    test("200: Responds with an array of article/articles filtered by topic query cats", () => {
```

Comment:
We shouldn't be doing a test for each valid topic. Just once valid topic with articles would be fine. However, we may want to tweak our other sad path tests to instead be:
Status 200, valid topic query, but has no articles responds with an empty array of articles, e.g. ?topic=paper.
Status 404, non-existent topic query, e.g. ?topic=bananas.